ui/app/services/pagination.js

```diff
@@ -34,16 +32,8 @@ export function keyForCache(query) {
   return JSON.stringify(cacheKeyObject);
 }
 
-export default class StoreService extends Store {
-  requestManager = new RequestManager();
-
-  constructor(args) {
-    super(args);
-    // If at some point we no longer need an extended store, we can remove the @ember-data/legacy-compat dep
-    // See: https://api.emberjs.com/ember-data/4.12/modules/@ember-data%2Frequest
-    this.requestManager.use([LegacyNetworkHandler]);
-    this.requestManager.useCache(CacheHandler);
-  }
+export default class Pagination extends Service {
```

Comment:
Name here seemed intuitive, but open to alternatives!

Reply:
I 💚 it.